sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/http/policy/TracingInfo.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.config.implementation.http.policy;
+
+import static com.azure.spring.cloud.config.implementation.AppConfigurationConstants.DEV_ENV_TRACING;
+import static com.azure.spring.cloud.config.implementation.AppConfigurationConstants.KEY_VAULT_CONFIGURED_TRACING;
+
+import com.azure.spring.cloud.config.implementation.HostType;
+import com.azure.spring.cloud.config.implementation.RequestTracingConstants;
+import com.azure.spring.cloud.config.implementation.RequestType;
+
+public class TracingInfo {
+
+    private boolean isDev = false;
+
+    private boolean isKeyVaultConfigured = false;
+
+    private int replicaCount = 0;
+
+    private FeatureFlagTracing featureFlagTracing;
+
+    public TracingInfo(boolean isDev, boolean isKeyVaultConfigured, int replicaCount) {
+        this.isDev = isDev;
+        this.isKeyVaultConfigured = isKeyVaultConfigured;
+        this.replicaCount = replicaCount;
+        this.featureFlagTracing = new FeatureFlagTracing();
+    }
+
+    public String getValue(boolean watchRequests) {
+        String track = System.getenv(RequestTracingConstants.REQUEST_TRACING_DISABLED_ENVIRONMENT_VARIABLE.toString());
+        if (track != null && !Boolean.valueOf(track)) {
```

Comment:
Could you please add unit test when the environment variable is?
1. null
2. empty string
3. "false"
4. "true"
5. "random string“